CPAN/Docker/README.md

```diff
@@ -11,8 +11,8 @@ Don't forget to adjust platform and base image versions to cover your needs.
 
 ```
 cd CPAN
-docker build --rm --platform=linux/amd64 -f "Docker/Dockerfile.fedora" -t slimservervendor:fedora .
-docker run --rm --platform=linux/amd64 -v `pwd`:/cpan slimservervendor:fedora ./buildme.sh
+podman build --rm --platform=linux/amd64 -f "Docker/Dockerfile.fedora" -t slimservervendor:fedora .
+podman run --rm --platform=linux/amd64 -v `pwd`:/cpan:Z localhost/slimservervendor:fedora ./buildme.sh
```

Comment:
What do the addition in the volume map stand for?

Reply:
The `:Z` basically means "make it work with SELinux" (which is enabled by default on Fedora).  https://docs.podman.io/en/latest/markdown/podman-run.1.html#volume-v-source-volume-host-dir-container-dir-options